utils/Linux/Linux.sh

```diff
@@ -1,11 +1,33 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
 
-export DISTRO=$(lsb_release -ds | cut -d " " -f 1)
+# Ensure OS is set (needed by the handoff path)
+export OS="${OS:-$(uname -s)}"
```

Comment:
this is exported from th main script (``installation.sh``)
```suggestion
export OS="${OS:-$(uname -s)}"
```
I am trying to enforce the poping out of possible errors in the flow. In this case, it would be hard to debug the scripts flow and exporting of variables if we have multiple nested export (especially if they are not initialized, i.e., the :-) 

I would also stick to the distro extraction from lsb_release
```suggestion
export DISTRO=$(lsb_release -ds | cut -d " " -f 1)
```
It is more maintainable and flexible (otherwise for every possible linux distro we should add another line in the case statement.